lib/takeaway.rb

```diff
@@ -0,0 +1,31 @@
+require 'order'
+require 'sms'
+
```

Comment:
I guess these are set up in your local environment to load with require? I haven't set this up yet in mine and it didn't work in irb so I changed it to require_realtive, but perhaps I should have made it work some other way?